css/contact.css

```diff
@@ -0,0 +1,131 @@
+#contact-button {
+  display: flex;
+  align-items: center;
+  border-radius: 8px;
+  padding: 20px;
+  position: relative;
+  top: 140px;
+  width: 150px;
+  height: 30px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  color: #6070ff;
+  border: none;
+}
+
+#contact-button:hover {
+  background-color: #2230d2;
+  color: #fff;
+}
+
+#contact {
+  background-image: url(/Images/contact\ form\ shapes-mobile@2x.png);
+  background-color: #6070ff;
+  background-repeat: no-repeat;
+  background-position-x: 120px;
+  background-position-y: 12px;
+  background-size: auto 757px;
+  border-radius: 100px 0% 0% 0%;
+  padding: 20px;
+  height: 773px;
+  position: relative;
+  bottom: 15vh;
+  display: flex;
+  flex-flow: column wrap;
+  width: 327px;
+  margin-top: 10vh;
+}
+
+#contact-background {
+  position: relative;
+  left: 30%;
+  height: 30vh;
+}
+
+form {
+  order: 3;
+  position: relative;
+  bottom: 50px;
+}
+
+#contact p {
+  position: relative;
+  width: 327px;
+  height: 102px;
+  top: 60px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 28px;
+  text-align: center;
+  color: #ebebff;
+}
+
+#contact h2 {
+  position: relative;
+  width: 100%;
+  height: 56px;
+  left: 5%;
+  top: 30px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #fff;
+  order: 1;
+}
+
+#contact input {
+  position: relative;
+  width: 90%;
+  height: 48px;
+  left: 0;
+  right: 0;
+  top: 100px;
+  margin-top: 20px;
+  bottom: 0;
+  background: #fff;
+  border: 1px solid #cfd8dc;
+  border-radius: 8px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 17px;
+  line-height: 20px;
+  display: flex;
+  align-items: center;
+  color: #172b4d;
+  padding-left: 10px;
+  order: 3;
+}
+
+#contact textarea {
+  position: relative;
+  top: 120px;
+  width: 90%;
+  height: 190px;
+  border-radius: 8px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #b3bac5;
+  padding-left: 10px;
+}
+
+.bottom {
+  order: 4;
+  position: relative;
+  top: 20vh;
+  left: 18vh;
+}
```

Comment:
- [ ] [OPTIONAL] You have done a great job adding button interactions, they are really important to enhance user experience however please make sure to implement button interaction according to the Figma design. Currently, your hover and presses state interaction have the same background (hover state should have a `background-color:#6070FF;`). Optionally you can always add `box shadow` if you feel like hover interaction makes your button blend into the background 

**_Figma design_** 

<img width="564" alt="Screenshot 2023-02-23 at 10 08 56 AM" src="https://user-images.githubusercontent.com/44798044/220825780-5ece94b0-b3a2-4dbe-877c-986643a484d3.png">